easybuild/easyconfigs/s/Sniffles/Sniffles-2.0.7-foss-2022a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'PythonBundle'
+
+name = 'Sniffles'
+version = '2.0.7'
+
+homepage = 'https://github.com/fritzsedlazeck/Sniffles'
+description = """A fast structural variant caller for long-read sequencing,
+ Sniffles2 accurately detect SVs on germline, somatic and population-level for PacBio and Oxford Nanopore read data."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
```

Comment:
Both dependencies are at GCCcore/GCC level, does Sniffles really need `foss`? Which components of the foss toolchain does it use?